3.2/s2i/bin/assemble

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shopt -s dotglob
+echo "---> Installing application source ..."
+ls -A /tmp/src/* &>/dev/null && mv /tmp/src/* ./
```

Comment:
@bparees Are files in `/tmp/src/` always owned by current images user? (does `s2i build` always set this?)
If not, `mv /tmp/src/* ./ 2>/dev/null` could hide useful error message. But it is true that `|| true` is more readable.

Anyway, thanks for suggestions.